api/shape/Arc.js

```diff
@@ -61,10 +57,7 @@ registerReifier('Arc', (geometry) => {
     })
       .scale(...scale)
       .move(...middle)
-      .move(...getAt(geometry))
-      .transform(getMatrix(geometry))
-      .setTags(geometry.tags)
-      .toGeometry();
+      .move(...getAt(geometry));
```

Comment:
How does .getAt() work?

Reply:
.getAt retrieves from .hasAt

Although I'm not so sure we really need hasAt().